src/main/java/com/google/sps/servlets/PostAnswerServlet.java

```diff
@@ -26,6 +26,7 @@
 import java.util.Date;
 import java.util.logging.Level;
 import java.util.logging.Logger;
+import javax.servlet.ServletException;
 import javax.servlet.annotation.WebServlet;
```

Comment:
Does it comes before the annotation.WebServlet import because of the uppercase? If not the order should be changed :)

Reply:
yes, in ASCII upper case characters come before lower case characters